modules/apps/headless-apio/media-object/media-object-apio-client-test/src/testIntegration/java/com/liferay/media/object/apio/client/test/MediaObjectApioTest.java

```diff
@@ -226,6 +272,212 @@ public void testDeleteDocument() {
 		);
 	}
 
+	@Test
+	public void testGetAdaptiveMediaFileFromImageInDocumentRepository()
+		throws IOException {
+
+		String imageFileName = "image.png";
+
+		_documentHref = ApioClientBuilder.given(
```

Comment:
Are we overriding a global variable _documentHref in a test?

Reply:
Exactly!